db/migrate/20200424122000_add_ancillary_metadata_to_portfolio.rb

```diff
@@ -0,0 +1,16 @@
+class AddAncillaryMetadataToPortfolio < ActiveRecord::Migration[5.2]
+  def up
+    create_table "ancillary_metadata".pluralize.to_sym do |t|
+      t.references "resource",   :polymorphic => true
+      t.jsonb      "statistics", :default => '{}'
+      t.bigint     "tenant_id"
+      t.timestamps
```

Comment:
I considered just added the `updated_at` column since I am throwing away the `created_at` when `metadata` is returned for the `Portfolio`.  I cannot think of any need for `created_at` on this table, but decided to leave it in and get feedback.

Thoughts?